src/west/app/project.py

```diff
@@ -124,10 +124,12 @@ def __init__(self):
 without -l, creates a workspace by cloning a manifest repository
 by URL.
 
-If cloning a repository:
+With -m, clones the repository at that URL and uses it as the
+manifest repository. If --mr is not given, the remote's currently
+active branch will be used, if it exists.
```

Comment:
Should we refer to HEAD instead "active"?

Reply:
HEAD feels like an implementation detail to me. "active"  is used in`git help clone` and `git help glossary` but is nowhere to be found in `git help checkout` where HEAD is used instead :-( git as usual...

As far as the server side is concerned, I believe the most usual name is neither: https://www.google.com/search?q=git+default+branch

Reply:
Right, I'm imitating git clone here.

Reply:
I searched more and I think we have a clear winner. `git clone` is surprisingly "wrong".
"Active" is much better than "default" for a local, developer clone. Not so much for a server.

https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-the-default-branch-name-for-your-repositories
https://docs.github.com/en?query=%22default+branch%22

https://docs.gitlab.com/ee/user/project/repository/branches/#default-branch
https://docs.gitlab.com/search/?query=%22default+branch%22


https://gerrit-review.googlesource.com/Documentation/project-configuration.html#default-branch

https://gerrit.googlesource.com/git-repo/+/50a81de2bc9f20%5E%21/
```python
def ResolveRemoteHead(self, name=None):
    """Find out what the default branch (HEAD) points to.

     `output = self.bare_git.ls_remote('-q', '--symref', '--exit-code', name, 'HEAD')`
```


Git itself uses a variety of names in the same paragraph to confuse the reader and "default" is one of them:


git help remote
```
       set-head
           Sets or deletes the default branch (i.e. the target of the symbolic-ref refs/remotes/<name>/HEAD) for the named
           remote. Having a default branch for a remote is not required, but allows the name of the remote to be specified in
           lieu of a specific branch. For example, if the default branch for origin is set to master, then origin may be
           specified wherever you would normally specify origin/master.
```

https://github.com/git/git/blob/master/Documentation/gitrepository-layout.txt

```

HEAD::
	A symref (see glossary) to the `refs/heads/` namespace
	describing the currently active branch.  It does not mean
	much if the repository is not associated with any working tree
	(i.e. a 'bare' repository), but a valid Git repository
	*must* have the HEAD file; some porcelains may use it to
	guess the designated "default" branch of the repository
	(usually 'master'). 
	
```